logicrunner/builtin/contract/member/member.go

```diff
@@ -197,9 +203,35 @@ func (m *Member) registerNodeCall(params map[string]interface{}) (interface{}, e
 
 	return m.registerNode(publicKey, role)
 }
-func (m *Member) addBurnAddressesCall(params map[string]interface{}) (interface{}, error) {
 
-	burnAddressesI, ok := params["burnAddresses"].([]interface{})
+func (m *Member) migrationAdminCall(params map[string]interface{}, nameMethod string) (interface{}, error) {
+	migrationAdminContract := migrationadmin.GetObject(foundation.GetMigrationAdmin())
+
+	switch nameMethod {
+	case "getInfo":
+		return migrationAdminContract.Info()
+	case "addBurnAddresses":
+		return m.addMigrationAddressesCall(params)
+
+	case "activateDaemon":
+		migrationDaemon, ok := params["reference"].(string)
```

Comment:
Better to create activateDaemonCall method and use it here

Reply:
why? I do not agree. This will add a lot of unnecessary lines

Reply:
Use the same handling way as in member.Call. 
Code should be similar.

Reply:
this will increase the amount of code

Reply:
Readable of code is more important then amount of code

Reply:
@sansaian I agree, it is better to put it in a separate method

Reply:
Done